modules/client/examples/03-tokenVoting-client/20-is-governance-token-compatible.ts

```diff
@@ -0,0 +1,35 @@
+/* MARKDOWN
+---
+title: Is governance token compatible
```

Comment:
```suggestion
title: Is TokenVoting compatible token
```
Not sure about the formatting style you are following for the titles

Please also rename the file to match the function name.